### PR TITLE
swaybg: add page

### PR DESCRIPTION
**Pull request by ivandimitrov8080**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

 swaybg version 1.2.0

### CODE REVIEW

**Review by leonvsc — APPROVED** [review]
LGTM! Thanks for your contribution.

File: pages/linux/swaybg.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# swaybg
+
+> Wallpaper tool for Wayland compositors.
+> More information: <https://github.com/swaywm/swaybg>.
+
+- Set an image as wallpaper:
+
+`swaybg -i "{{path/to/image}}"`
```
Comment (gutjuri) [inline pages/linux/swaybg.md:8]:
```suggestion
`swaybg -i {{path/to/image}}`
```

In the past, we've omitted the quotation marks

File: pages/linux/swaybg.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Set the wallpaper mode:
+
+`swaybg -i "{{path/to/image}}" -m {{stretch | fit | fill | center | tile | solid_color}}`
```
Comment (gutjuri) [inline pages/linux/swaybg.md:12]:
```suggestion
`swaybg -i {{path/to/image}} -m {{stretch|fit|fill|center|tile|solid_color}}`
```

In the other pages, we do not have spaces in such enumerations

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks for you contribution! I have some suggestions.

File: pages/linux/swaybg.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`swaybg -i {{path/to/image}}`
+
+- Set the wallpaper mode:
+
+`swaybg -i {{path/to/image}} -m {{stretch|fit|fill|center|tile|solid_color}}`
+
+- Set a static color as a wallpaper:
+
+`swaybg -c {{"#rrggbb"}}`
```
Comment (kbdharun) [inline pages/linux/swaybg.md:16]:
```suggestion
- Set an [i]mage as wallpaper:

`swaybg --image {{path/to/image}}`

- Set the wallpaper [m]ode:

`swaybg --image {{path/to/image}} --mode {{stretch|fit|fill|center|tile|solid_color}}`

- Set a static [c]olor as a wallpaper:

`swaybg --color {{"#rrggbb"}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page. We prefer long options throughout the pages (in most cases) and we highlight short options using mnemonics whenever applicable.

File: pages/linux/swaybg.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# swaybg
+
+> Wallpaper tool for Wayland compositors.
+> More information: <https://github.com/swaywm/swaybg>.
```
Comment (acuteenvy) [inline pages/linux/swaybg.md:4]: The README doesn't contain a lot of info, I think the man page is better. It's even referenced in the README:
> See the man page, swaybg(1), for instructions on using swaybg.

```suggestion
> More information: <https://github.com/swaywm/swaybg/blob/master/swaybg.1.scd>.
```

File: pages/linux/swaybg.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`swaybg --image {{path/to/image}}`
+
+- Set the wallpaper [m]ode:
+
+`swaybg --image {{path/to/image}} --mode {{stretch|fit|fill|center|tile|solid_color}}`
+
+- Set a static [c]olor as a wallpaper:
+
+`swaybg --color {{"#rrggbb"}}`
```
Comment (acuteenvy) [inline pages/linux/swaybg.md:16]:
```suggestion
- Set the wallpaper to an [i]mage:

`swaybg --image {{path/to/image}}`

- Set the wallpaper [m]ode:

`swaybg --image {{path/to/image}} --mode {{stretch|fit|fill|center|tile|solid_color}}`

- Set the wallpaper to a static [c]olor:

`swaybg --color {{"#rrggbb"}}`
```

**Review by acuteenvy — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**